src/test/java/org/broadinstitute/consent/http/resources/LibraryCardResourceTest.java

```diff
@@ -0,0 +1,218 @@
+package org.broadinstitute.consent.http.resources;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.when;
+
+import com.google.api.client.http.HttpStatusCodes;
+import com.google.gson.Gson;
+import java.util.Collections;
+import javax.ws.rs.core.UriInfo;
+import javax.ws.rs.core.UriBuilder;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.core.Response;
+import java.util.List;
+import java.util.Date;
+import org.jdbi.v3.core.statement.UnableToExecuteStatementException;
+import org.broadinstitute.consent.http.enumeration.UserRoles;
+import org.broadinstitute.consent.http.models.AuthUser;
+import org.broadinstitute.consent.http.models.User;
+import org.broadinstitute.consent.http.models.UserRole;
+import org.broadinstitute.consent.http.models.LibraryCard;
+import org.broadinstitute.consent.http.service.UserService;
+import org.broadinstitute.consent.http.service.LibraryCardService;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.postgresql.util.PSQLException;
+import org.postgresql.util.PSQLState;
+
+public class LibraryCardResourceTest {
+  private final AuthUser authUser = new AuthUser("test@test.com");
+  private final List<UserRole> adminRoles = Collections.singletonList(
+    new UserRole(UserRoles.ADMIN.getRoleId(), UserRoles.ADMIN.getRoleName())
+  );
```

Comment:
We do this so much, it might be nice to extract them into constants in a base test class.